src/devices/eglo.ts

```diff
@@ -47,6 +47,15 @@ const definitions: Definition[] = [
             'green', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 'brightness_move_down', 'brightness_stop',
             'recall_1', 'color_temperature_step_up', 'color_temperature_step_down'])],
     },
+    {
+        zigbeeModel: ['TLSR82xx'],
```

Comment:
Please use a `fingerprint` here (since there are many devices using this modelID). You can grab the details from the `data/database.db` file, example:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/fb90742b8c59300ec1bd47eec7e4d441d2fd85da/src/devices/awox.ts#L45